app/views/products/index.html.erb

```diff
@@ -1,7 +1,8 @@
 <h1>Home Page</h1>
 
-
-<%= link_to "Create new product", new_admin_product_path %>
+<% if current_user %>
```

Comment:
Hey Hadi! Code looks great! Just to clarify, users should not be able to create new product right? So this link should only be accessible in the admin/products index.html.erb.